org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/debugger/server/rmi/XDebugStackFrameVariable.xtend

```diff
@@ -21,7 +20,8 @@ class XDebugStackFrameVariable implements Serializable {
 	}
 	
 	def dispatch asRemoteValue(WollokObject object) { new XWollokObjectDebugValue(variable.name, object) }
-	def dispatch asRemoteValue(WollokList list) { new XWollokListDebugValue(list) }
+	// TODO: after changing lists to wollokobjects
+//	def dispatch asRemoteValue(WollokList list) { new XWollokListDebugValue(list) }
 	def dispatch asRemoteValue(Object o) { new XDebugValue(o.toString) }
```

Comment:
Yeap, good that you noticed it. I forgot about that. I broke the special visualization of lists in the debugger UI.
I need to fix that. But I guess that it could be after merging